src/drives/worksheets_used_range/request.rs

```diff
@@ -0,0 +1,22 @@
+// GENERATED CODE
+
+use crate::api_default_imports::*;
+
+resource_api_client!(
+    WorksheetsUsedRangeApiClient,
+    ResourceIdentity::WorksheetsUsedRange
+);
+
+impl WorksheetsUsedRangeApiClient {
+    get!(
+        doc: "Invoke function usedRange",
+        name: workbook_worksheet,
+        path: "/usedRange()"
+    );
+    get!(
+        doc: "Invoke function usedRange",
+        name: workbook_worksheet,
+        path: "/usedRange(valuesOnly={{id}})",
+        params: values_only
+    );
+}
```

Comment:
Both function names here are the same. So this fails to compile.

But, this is a case where the method names need to be changed instead of generating a separate client. 

See my other comment on this.

Reply:
done